<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonFloor1">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>24</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFloor2">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>185</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>24</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFloor3">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>24</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>10</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>50</number>
    </property>
    <property name="autoRepeatInterval">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>10</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>50</number>
    </property>
    <property name="autoRepeatInterval">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelFloor">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>25</y>
      <width>60</width>
      <height>66</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color:rgb(0, 0, 0);color:rgb(239, 239, 239);}</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelUpDown">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>25</y>
      <width>60</width>
      <height>66</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi Cond</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color:rgb(0, 0, 0);color:rgb(239, 239, 239);}</string>
    </property>
    <property name="text">
     <string>â–²</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelBackGround">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color:rgb(0, 0, 0);color:rgb(239, 239, 239);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelDoorRight">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>78</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color:rgb(0,0,0);color:rgb(239, 239, 239)}</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelDoorLeft">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>78</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color:rgb(0,0,0);color:rgb(239, 239, 239)}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <zorder>labelBackGround</zorder>
   <zorder>pushButtonFloor1</zorder>
   <zorder>pushButtonFloor2</zorder>
   <zorder>pushButtonFloor3</zorder>
   <zorder>pushButtonOpen</zorder>
   <zorder>pushButtonClose</zorder>
   <zorder>labelFloor</zorder>
   <zorder>labelUpDown</zorder>
   <zorder>labelDoorRight</zorder>
   <zorder>labelDoorLeft</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
